### ISSUE TITLE
Unable to run docker run hello-world because of SE-Linux

### ISSUE THREAD

**Issue by aanm** [docker/docker#14373]
```
$ docker run hello-world
Unable to find image 'hello-world:latest' locally
latest: Pulling from hello-world
a8219747be10: Pull complete 
91c95931e552: Already exists 
hello-world:latest: The image you are pulling has been verified. Important: image verification is a tech preview feature and should not be relied on to provide security.
Digest: sha256:aa03e5d0d5553b4c3473e89c8619cf79df368babd18681cf5daeb82aab55838d
Status: Downloaded newer image for hello-world:latest
permission denied
Error response from daemon: Cannot start container e7fe20431c30ae7b96ae0078ed6dfc03d4aa9276874aa3934301863610d7e1d7: [8] System error: permission denied
```

```
$ cat /etc/sysconfig/docker
# /etc/sysconfig/docker

# Modify these options if you want to change the way the docker daemon runs
OPTIONS='--selinux-enabled --storage-driver=devicemapper'
DOCKER_CERT_PATH=/etc/docker

# Enable insecure registry communication by appending the registry URL
# to the INSECURE_REGISTRY variable below and uncommenting it
# INSECURE_REGISTRY='--insecure-registry '

# On SELinux System, if you remove the --selinux-enabled option, you
# also need to turn on the docker_transition_unconfined boolean.
# setsebool -P docker_transition_unconfined

# Location used for temporary files, such as those created by
# docker load and build operations. Default is /var/lib/docker/tmp
# Can be overriden by setting the following environment variable.
# DOCKER_TMPDIR=/var/tmp

# Controls the /etc/cron.daily/docker-logrotate cron job status.
# To disable, uncomment the line below.
# LOGROTATE=false
```
```
$ sudo docker run hello-world
permission denied
Error response from daemon: Cannot start container 2ccf2288382d3350c6bc2e7012d63c0da52b8e442120da786f5e9e86ef6c21ca: [8] System error: permission denied
$ sudo setenforce 0
$ sudo docker run hello-world
Hello from Docker.
This message shows that your installation appears to be working correctly.

To generate this message, Docker took the following steps:
 1. The Docker client contacted the Docker daemon.
 2. The Docker daemon pulled the "hello-world" image from the Docker Hub.
    (Assuming it was not already locally available.)
 3. The Docker daemon created a new container from that image which runs the
    executable that produces the output you are currently reading.
 4. The Docker daemon streamed that output to the Docker client, which sent it
    to your terminal.

To try something more ambitious, you can run an Ubuntu container with:
 $ docker run -it ubuntu bash

For more examples and ideas, visit:
 http://docs.docker.com/userguide/
```

```
[aanm@localhost docker]$ uname -a
Linux localhost.localdomain 4.0.6-300.fc22.x86_64 #1 SMP Tue Jun 23 13:58:53 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux
```
```
[aanm@localhost docker]$ docker version
Client:
 Version:      1.8.0-dev
 API version:  1.20
 Go version:   go1.4.2
 Git commit:   71f8ca3
 Built:        Thu Jul  2 21:11:07 UTC 2015
 OS/Arch:      linux/amd64

Server:
 Version:      1.8.0-dev
 API version:  1.20
 Go version:   go1.4.2
 Git commit:   71f8ca3
 Built:        Thu Jul  2 21:11:07 UTC 2015
 OS/Arch:      linux/amd64
```
```
$ docker info
Containers: 7
Images: 48
Storage Driver: devicemapper
 Pool Name: docker-253:1-262724-pool
 Pool Blocksize: 65.54 kB
 Backing Filesystem: extfs
 Data file: /dev/loop0
 Metadata file: /dev/loop1
 Data Space Used: 2.148 GB
 Data Space Total: 107.4 GB
 Data Space Available: 16.19 GB
 Metadata Space Used: 3.539 MB
 Metadata Space Total: 2.147 GB
 Metadata Space Available: 2.144 GB
 Udev Sync Supported: false
 Deferred Removal Enabled: false
 Data loop file: /var/lib/docker/devicemapper/devicemapper/data
 Metadata loop file: /var/lib/docker/devicemapper/devicemapper/metadata
 Library Version: 1.02.82-git (2013-10-04)
Execution Driver: native-0.2
Logging Driver: json-file
Kernel Version: 4.0.6-300.fc22.x86_64
Operating System: Fedora 22 (Twenty Two)
CPUs: 8
Total Memory: 15.46 GiB
Name: localhost.localdomain
ID: UTTO:O2B2:3TZB:7YSQ:VWV5:C5WN:PSYJ:PMCZ:R5O3:N7QU:3G2M:DKVA
```

**Comment by rhatdan** [comment]
What AVC's are you getting?    Are you sure docker is runing as docker_t?  Do you have docker-selinux installed?

**Comment by aanm** [comment]
@rhatdan It seems the problem is solved on docker 1.8.1 thanks.
